tests/features/warmshowers/create_account.feature

```diff
@@ -0,0 +1,55 @@
+#language:en
+
+Feature: I can create an account
+	In order to become a WarmShowers member
+	As a new user
+	I can create an account
+
+Scenario Outline: As a new user, I can view the account creation page in my language

```

Comment:
We have a very similar scenario in the header_navigation feature. I don't think we need this one.